config/samples/testing_v1alpha1_testsuite.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: testing.kyma-project.io/v1alpha1
+kind: TestSuite
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: testsuite-sample
+spec:
+  # Add fields here
+  concurrency_level: 2
+  test_names_selector:
```

Comment:
How about label selectors instead? https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors

Reply:
IMO we can extend API later to support also label selector. Why I like test_names_selector:
- you can specify only one test to run, without introducing artificial labels for it
- we can use test_names_selectors to run only failed tests from another test suite. So I have an execution that failed, I can fetch names of the tests that failed and execute only them.